### PR TITLE
[typescript] #2488

### PR DESCRIPTION
**Pull request by ksamborski**
A fix for #2488 for better variables parsing. Especially when they contain regexes (/.../) or division (/).

### CODE REVIEW

File: Units/parser-typescript.r/ts-const.d/input.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import * as three from "three";
```
Comment (masatake) [inline Units/parser-typescript.r/ts-const.d/input.ts:1]: I would like you to write the source of the test input file like
```
// Taken from issue #2488 submitted by @sparkcanon
```

(Fix the example if my English is wrong.)

Comment (ksamborski) [reply Units/parser-typescript.r/ts-const.d/input.ts:1]: Good point, thanks! I've updated the test file.

Comment (masatake) [reply Units/parser-typescript.r/ts-const.d/input.ts:1]: Updated in c8cb4a4 ?
As far as seeing https://github.com/universal-ctags/ctags/pull/2493/files, the test input is not changed.

Comment (ksamborski) [reply Units/parser-typescript.r/ts-const.d/input.ts:2]: yeah, I was doing `git commit --amend` and forgot to run `git add` first. It should be ok now.

**Comment by sparkcanon** [comment]
I tested your branch, @ksamborski and noticed that more tags are now listed.
```

src/index.ts,561
const scene: three.Scene,scene3,33
  camera: three.PerspectiveCamera,camera4,59
  renderer: three.WebGLRenderer,renderer5,94
const setupScene = (): void => {setupScene19,401
const createCube = (): three.Mesh => {createCube37,886
const createCubeArr = (): void => {createCubeArr50,1275
const createWall = (): void => {createWall67,1702
const animate = (): void => {animate78,1999
```

Strangely, some other tags are not recognised such as,
`Units/parser-typescript.r/ts-const.d/input.ts` line 42
`Units/parser-typescript.r/ts-const.d/input.ts` line 43
`Units/parser-typescript.r/ts-const.d/input.ts` line 52
`Units/parser-typescript.r/ts-const.d/input.ts` line 27

**Comment by sparkcanon** [comment]
> @sparkcanon yes, but currently the body of lambda functions is ignored. I will look into that in the future.

Makes sense, thank you.

**Pull request merged**